lib/handlers/shared.ts

```diff
@@ -84,6 +83,7 @@ export const DEFAULT_ROUTING_CONFIG_BY_CHAIN = (chainId: ChainId): AlphaRouterCo
         forceCrossProtocol: false,
       }
     case ChainId.ZKSYNC:
+    case ChainId.BASE:
```

Comment:
@msvstj not sure about this change, let me know if it should be reverted!

Reply:
Yep, this should be reverted. In general default values for base should not be changed (except RPC - good catching that part)!